PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidMobile.java

```diff
@@ -22,9 +22,9 @@
 
 public class PrebidMobile {
 
-    public static final int TIMEOUT_MILLIS = 2_000;
+    private static final int TIMEOUT_MILLIS = 2_000;
 
-    static int timeoutMillis = TIMEOUT_MILLIS; // by default use 10000 milliseconds as timeout
+    public static int timeoutMillis = TIMEOUT_MILLIS; // by default use 10000 milliseconds as timeout
```

Comment:
If the objective here is to providing publisher's code access to the `timeoutMillis`, then why not expose it using the public facing API. 
For reference:
```
    public static int getTimeoutMillis() {
        return timeoutMillis;
    }

    public static void setTimeoutMillis(int timeoutMillis) {
        PrebidMobile.timeoutMillis = timeoutMillis;
    }
```

Also, please correct the default value written in this comment - `// by default use 10000 milliseconds as timeout`

Reply:
Hi @avohraa , I have made the change. Thanks,